envs/prod/env.hcl

```diff
@@ -0,0 +1,4 @@
+locals {
+  project_id  = "my-prod-project"
```

Comment:
Is defined as `get_env("VERTEX_PROJECT_ID")` in `envs/dev/env.hcl`, is there a reason its hard coded here?

Reply:
I guess the idea is that if you are just wanting to try out the repo, spin up the environment by running terragrunt locally. If you want the full-fledged multi-project deployment, you would run this in CI/CD

Prod project would never be the same as the env variable set in env.sh for local development